src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -672,6 +672,15 @@ internal async Task SendInvocationRequest(ScriptInvocationContext context)
                     return;
                 }
 
+                foreach ((string, DataType, object) element in context.Inputs)
+                {
+                    if (element.Item3.GetType() == typeof(ParameterBindingData) || element.Item3.GetType() == typeof(ParameterBindingData[]))
+                    {
+                        _workerChannelLogger.LogInformation("Function '{functionName}' binding to ParameterBindingData for parameter '{name}', datatype '{datatype}' and binding '{binding}'", context.FunctionMetadata.Name, element.Item1, element.Item2, element.Item3);
```

Comment:
Question: Do we need to log this to functionslogs as well? Is logging to FunctionsMetrics not enough? We probably are going to run aggregate queries against the metrics table. Don't we?

Reply:
Checking with Mathew in another comment and will remove log if not needed